File: 35-search-insert-position/35-search-insert-position.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
        
    
    int counter=0;
    for( int i=0 ;i < nums.size()  ; i++ )
        {
            if (nums[i] < target)
                ++counter;
            else
                break;
        }
    
    return counter;
        
    }
};

File: 35-search-insert-position/35-search-insert-position.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert = function(nums, target) {
    let counter=0;
   
    for( let i=0 ;i < nums.length  ; i++ )
        {
            if (nums[i] < target)
                ++counter;
            else
                break;
        }
    
    return counter;
        
};